src/test/java/io/logz/MockLogzioBulkListener.java

```diff
@@ -149,10 +153,63 @@ public boolean checkForLogExistence(String token, String type, String loggerName
                 if (!jsonObject.get("message").getAsString().equals(message)) {
                     found = false;
                 }
+
+                if (checkHostname) {
```

Comment:
I think this should be done on the JUnit. the mock should only hold the received msgs and serve them via getter method.